### DISCUSSION TITLE
promise ring issue

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7879] [QA - tech support + general]
soo..i hit 400 affection, but it reset to 396 so i gave it before it reset then monika declined it bc then it reset but now im at 412 but when i try and gift it again she doesn't notice it. helppp!!!

**Comment by KritRom** [comment]
what do you mean by "reset"?

**Comment by ghost** [comment]
> what do you mean by "reset"?

i mean,there is a affection cap so i hit the limit of affection i could get

**Comment by KritRom** [comment]
> > what do you mean by "reset"?
> 
> i mean,there is a affection cap so i hit the limit of affection i could get

ah okay, just drop your `aff_log.txt` file and I'll take a look

**Comment by ghost** [comment]
> > > what do you mean by "reset"?
> > 
> > 
> > i mean,there is a affection cap so i hit the limit of affection i could get
> 
> ah okay, just drop your `aff_log.txt` file and I'll take a look

ohh ok

**Comment by ghost** [comment]
> > > > what do you mean by "reset"?
> > > 
> > > 
> > > i mean,there is a affection cap so i hit the limit of affection i could get
> > 
> > 
> > ah okay, just drop your `aff_log.txt` file and I'll take a look
> 
> ohh ok

wait im on macbook

**Comment by KritRom** [comment]
Okay, just drop the last 10 lines of the terminal thing

**Comment by ghost** [comment]
> > > > > what do you mean by "reset"?
> > > > 
> > > > 
> > > > i mean,there is a affection cap so i hit the limit of affection i could get
> > > 
> > > 
> > > ah okay, just drop your `aff_log.txt` file and I'll take a look
> > 
> > 
> > ohh ok
> 
> wait im on macbook

heres a screenshot:

![Screen Shot 2021-09-22 at 9 43 19 AM](https://user-images.githubusercontent.com/82217573/134365688-12658a70-6639-4b10-b3c4-3669fd8244de.png)

**Comment by Booplicate** [comment]
It didn't reset, it didn't get over 400 in the first place. Since you already tried to gift the ring today, you can't do it again until tomorrow.
